src/deepak/assignment24/Program_2.java

```diff
@@ -0,0 +1,41 @@
+/*program 2 : sum of all numbers in a give string 
+input: te123ch9kj8 
+output : 123 + 9 + 8 = 140 */
+
+package deepak.assignment24;
+
+public class Program_2 {
+	
+	void sumOfAllNumbers(String input) {
+		String s = "";
+		int sum = 0;
+		for (int i = 0; i < input.length(); i++) {
+			char ch = input.charAt(i);
+
+			if (Character.isDigit(ch)) {
+				s = s + ch;
+
+			} else if (s.isEmpty()) {
+				continue;
```

Comment:
This is good,

I feel you also check this code as well,
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/752/files